pythonlib/src/mill/pythonlib/PythonModule.scala

```diff
@@ -22,9 +22,21 @@ trait PythonModule extends Module with TaskModule { outer =>
   def sources: T[Seq[PathRef]] = Task.Sources { millSourcePath / "src" }
 
   /**
-   * The script to run. This file may not exist if this module is only a library.
+   * The folders where the resource files for this module live.
    */
-  def script: T[PathRef] = Task.Source { millSourcePath / "src" / "main.py" }
+  def resources: T[Seq[PathRef]] = Task.Sources { millSourcePath / "resources" }
+
+  /**
+   * All Python source files in this module, recursively discovered from the source directories.
+   */
+  def allSourceFiles: T[Seq[PathRef]] = Task {
```

Comment:
Do you know if this task will be used in `PythonModule` in the future? Right now I think it's only used in your example, so I'd suggest to remove it from here unless we can anticipate a use-case to keep it here

Reply:
btw for now its only used for example but may be in future it come to work (not sure)

Reply:
Let's remove it from `PythonModule` for now then. I think a good principal to follow is to keep the modules in `pythonlib` minimal, since that's what we'll need to maintain. It's too early to see what will be used and we can always move this task definition back into here if we need to.

Reply:
ok sir i will !

could you please once revisit the replies as some of them need attention

Reply:
FIXED!